### PR TITLE
Add embark-mixed-indicator with delayed display of the vindicator

### LINKED ISSUE #296: Optional delay for the verbose indicator display?
As the title suggests, this is about making the verbose indicator more adaptable for those who normally know what action they will take.  An optional delay, which could be set to 0 by default, would help with a workflow where if, say, I wait for 1 second then I must have forgotten what key/action I want, so the detailed buffer comes to offer help.

At any rate, thanks for the new features!

### PR DESCRIPTION
**Pull request by minad**
Fixes #296. For full configurability combine this with the flexible display by @jaor in #297!

### CODE REVIEW

File: embark.el [hunk 1]
```diff
@@ -237,6 +237,10 @@ Embark comes with two such indicators:
   detailed information including key bindings and the first line
   of the docstring of the commands they run.
 
+- `embark-mixed-indicator', which combines the minimal and the
+  verbose indicator. The verbose popup is shown after
+  `embark-mixed-indicator-delay' seconds.
```
Comment (protesilaos) [inline embark.el:242]: Maybe the sentence starting with "The verbose popup..." needs to be on its own line?

Also, I think the compiler complains about a single space after a period.

Comment (oantolin) [reply embark.el:242]: I'll take care of those after merging, @protesilaos.

**Comment by oantolin** [comment]
Should the mixed one really be the default?

**Comment by protesilaos** [comment]
> Should the mixed one really be the default?

Seeing as this will benefit new and potentially unsuspecting users the most, I would go with the verbose.

**Comment by oantolin** [comment]
That was my thinking too, @protesilaos.

**Comment by oantolin** [comment]
Although, if your problem is that you are a new user and don't know what actions are available you'll probably hesitate and then the mixed indicator will pop up anyway.

**Comment by protesilaos** [comment]
Yes, indeed.

**Comment by protesilaos** [comment]
By the way:

```elisp
(defcustom embark-mixed-indicator-delay 0.5
  "Time in seconds after which the verbose indicator is shown."
  :type '(choice (const :tag "No delay" 0)
                 (number :tag "Delay in seconds")))
```

Is this intended only for the mixed style?  Or would it apply to the verbose as well?

**Comment by oantolin** [comment]
Only for the mixed indicator. The verbose indicator has no delay. I like it that way because I find the mixed indicator strictly superior to a delayed verbose one: before the delay is over, the mixed one shows you the minimal indicator which does provide very useful information: how many other targets there are at point.

**Comment by protesilaos** [comment]
I see.  Thanks!

**Pull request merged**